managers/tests/test_views.py

```diff
@@ -342,3 +344,73 @@ def test_remove_task_activities_view_should_change_task_status_to_not_active(sel
         self.client.post(self.url)
 
         self.assertIn(self.active_task_activity, TaskActivityType.objects.filter(is_active=False))
+
+
+class ExpectedHoursBaseTests(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.current_time = timezone.now()
+        self.user = UserFactory(
+            email="testuser@codepoets.it",
+            first_name="John",
+            last_name="Doe",
+            user_type=CustomUser.UserType.ADMIN.name,
+            is_active=True,
+        )
+
+        self.project = ProjectFactory()
+        self.client.force_login(self.user)
+        self.project.members.add(self.user)
+
+
+class ProjectExpectedHoursViewTests(ExpectedHoursBaseTests):
+    def setUp(self):
+        super().setUp()
+        self.url = reverse(
+            "project-expected-hours",
+            kwargs={"pk": self.project.pk, "year": self.current_time.year, "month": self.current_time.month},
+        )
+
+    def test_project_expected_hours_view_should_display_expected_hours_for_project_on_get(self):
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, ProjectExpectedHoursView.template_name)
+
```

Comment:
I think that you should also create expected hours for user and check that proper data are displayed in template

Reply:
Please also add test that to check that only authenticated users are able to view this page. I am not sure how this should work, bu also only manager/superuser should be able to view this page if I am not wrong, so please also add test for that.

Reply:
Maybe this is not related now, but @kbeker what do you think to add another test for removing user from project case? Should "archive" expected hours still display in template?

Reply:
1) Added new view/page that displays expected hours for user. 
2) Added tests